src/runtime/string.py

```diff
@@ -0,0 +1,17 @@
+from ltypes import i32, f64, f32, i64, ccall, c32, c64
+
+whitespace: str = ' \t\n\r\v\f'
+ascii_lowercase: str= 'abcdefghijklmnopqrstuvwxyz'
+ascii_uppercase: str = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+digits: str = '0123456789'
+hexdigits: str = digits + 'abcdef' + 'ABCDEF'
+octdigits: str = '01234567'
+punctuation: str = "#$%&'()*+,-./:;<=>?@[\]^_`{|}~"
+
+def capitalize(s: str) -> str:
+    """
+    Return a copy of the string with its first character capitalized and the rest lowercased.
+    """
+    result: str
+    result = s[0].upper() + s[1:]
```

Comment:
Right now, `.upper()` won't work. We should implement this in the library.

Reply:
Shall I wait for the implementation of .upper() or proceed via other techniques like adding to its ASCII value to convert to capital?

Reply:
You can implement upper as a function in `string.py` for now.

Reply:
@certik @Madhav2310 I came across this while surfing another `string` issue . I have implemented upper as a function 
```
def upper1(s):        # plain python 
    alphabet = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ'
    result = ''
    for x in s:
        for pos in range(52):
            if alphabet[pos] == x:
                i = pos
        if x not in alphabet or i>=26:
            result += x
        else:
            result += alphabet[i+26]
    return result

def upper2(s):      # using some inbuilt functions like ord() , chr()
    result = ''
    for char in s:
        if ord(char) >= 97 and ord(char) <=122 :
            result += chr(ord(char) - 32)
        else :
            result += char
    return result
```
Adding related functions like `lower`,`islower` ,`isupper` wouldn't be difficult either. Should I go ahead with this ? I think these can be updated in `lpython_builtin.py` or simply make a new file `runtime/string.py` which would also support this PR !

Reply:
@faze-geek yes, go ahead and implement those, and add tests.

Reply:
@faze-geek Thank you for your inputs! I'll go ahead and commit these functions with this PR. Do tell if you come up with further additions. 
cc: @certik @czgdp1807